src/Security/Identity.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of letswifi; a system for easy eduroam device enrollment
+ * Copyright: 2023, Paul Dekkers, SURF <paul.dekkers@surf.nl>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+namespace App\Security;
+
+use App\Entity\Contact;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+class Identity implements UserInterface
```

Comment:
Zoals hierboven, als dit voor de saml integratie si, wellicht even in een eigen namespace.